src/App.tsx

```diff
@@ -4,36 +4,123 @@ import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 import { UserDetailsProvider } from './context/UserDetailsContext';
 import Home from './screens/Home';
-import Post from './screens/Post';
 import LoginForm from './screens/LoginForm';
 import MenuBar from './screens/MenuBar';
+import Post from './screens/Post';
 import SignupForm from './screens/SignupForm';
 
 import 'bootstrap/dist/css/bootstrap.min.css';
+import './fonts.css' /* as createGlobalStyle is not importing fonts*/
+import { createGlobalStyle } from 'styled-components'
+
+
+export const GlobalStyle = createGlobalStyle`
```

Comment:
We should create a component outside of Apps so that we don't clutter it. Example: https://github.com/polkadot-js/apps/blob/master/packages/apps/src/Apps.tsx
(can be done in a separate PR)